tsl/src/compression/compression.c

```diff
@@ -2565,7 +2569,11 @@ process_predicates(Chunk *ch, CompressionSettings *settings, List *predicates, L
 					continue;
 
 				if (!IsA(expr, Const))
-					continue;
+				{
+					expr = (Expr *) estimate_expression_value(&root, (Node *) expr);
```

Comment:
Currently this means we're able to evaluate the stable functions in WHERE conditions, when it is on one side of a supported operator, right?

Reply:
correct, all the sqlvaluefunction (current_timestamp, current_user, ...) will also be supported with this